### ISSUE TITLE
Common file suffixes such as .out are classified as filetype tex

### ISSUE THREAD

**Issue by ghgary** [vim/vim#14722]
### Steps to reproduce

1. Use `strace -o strace.out someprogram` to create an strace log file named strace.out.
2. Open strace.out with vim: `vim --clean strace.out`.
3. The syntax coloring is wrong and the 'filetype` is tex.

### Expected behaviour

I expect the 'filetype' to be strace, as detected and set at line 378 of $VIMRUNTIME/autoload/dist/script.vim.

### Version of Vim

9.1.377

### Environment

Operating system: Ubuntu 20.04

### Logs and stack traces

```shell
The problem was introduced at patch 9.1.0289, "filetype: some TeX files are not recognized," wherein a whole slew of filename suffixes was added to the pattern for detecting TeX files at what is now line 2416 of $VIMRUNTIME/filetype.vim. There were some other suffixes that I'm pretty sure aren't unique to TeX, but I'm sure that .out does not belong there.

Removing that one suffix from that line restores correct behavior in the example above.
```

**Comment by zeertzjq** [comment]
Fixed by https://github.com/vim/vim/commit/e35478bc9d48189322432248105d3b24e0efb3d0

**Comment by zeertzjq** [comment]
Fixed by [patch 9.1.0383](https://github.com/vim/vim/commit/e35478bc9d48189322432248105d3b24e0efb3d0)

**Comment by ghgary** [comment]
Fixed four days ago. That'll teach me to update to the very latest version before reporting a bug. Thank you.

**Comment by ghgary** [comment]
Already fixed by patch 9.1.0383 and fix verified.
